exercises/alphametics/example.tt

```diff
@@ -3,16 +3,37 @@ gem 'minitest', '>= 5.0.0'
 require 'minitest/autorun'
 require_relative 'alphametics'
 
-# Test data version:
-# <%= sha1 %>
-class AlphameticsTest < Minitest::Test<% test_cases.each do |test_case| %>
+# Test data version: <%= sha1 %>
+class AlphameticsTest < Minitest::Test
+<% test_cases.each do |test_case| %>
   def <%= test_case.test_name %>
     <%= test_case.skipped %>
-    expect = <%= test_case.expect %>
-    actual = <%= test_case.work_load %>
-    assert_equal(expect, actual)
+    input = <%= test_case.input %>
+    expected = <%= test_case.expect %>
+    <%= test_case.workload %>
   end
 <% end %>
+  # These tests have been commented out due their long runtime.  If you are
+  # interested in optimsing your solution for speed these are a good tests to
+  # try.
+  #
```

Comment:
The 'long' tests are ignored by the test case generator but are hardcoded here for reference. 
It would also be possible (but more complicated) to generate them (I think I forgot that's what I started out trying to do.)